Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -488,7 +492,8 @@ def calculate_fetch_start_time(last_fetch, first_fetch):
     if last_fetch is None:
         if not first_fetch:
             first_fetch = '3 days'
-        first_fetch_time = date_to_timestamp(first_fetch)
+        first_fetch_dt = parse(first_fetch).replace(tzinfo=utc)
+        first_fetch_time = int(time.mktime(first_fetch_dt.timetuple()))
```

Comment:
```suggestion
        first_fetch_time = int(first_fetch_dt.timestamp())
```

This should do the trick and be more readable.